piccolo/columns/base.py

```diff
@@ -318,6 +318,9 @@ def __init__(
             }
         )
 
+        if (primary is True) and (key is False):
```

Comment:
Is there a case where `key` can be True and `primary` False? If they always should match then maybe we can turn them into a single field?

Reply:
You're right, it would make more sense if there was just a `primary_key` argument, rather than `primary` and `key`. Honestly, I'm not sure why I did it like this originally.

I'll try changing it, but will have to tread carefully in case it effects migrations.